tree/dataframe/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+############################################################################
+# CMakeLists.txt file for building ROOT tree/dataframe package
+# @author Danilo Piparo CERN, Pere Mato CERN
+############################################################################
+
+if (ARROW_FOUND)
+  include_directories(${ARROW_INCLUDE_DIR})
+endif()
+
+ROOT_GLOB_HEADERS(dictHeaders inc/*.h inc/ROOT/*.hxx)
+
+# this header is deprecated
+list(REMOVE_ITEM dictHeaders ${CMAKE_SOURCE_DIR}/tree/dataframe/inc/ROOT/TResultProxy.hxx)
+
+ROOT_GLOB_SOURCES(sources src/*.cxx)
+
+set(DATAFRAME_DEPENDENCIES Imt)
+
+if(NOT ARROW_FOUND)
+  list(REMOVE_ITEM dictHeaders ${CMAKE_CURRENT_SOURCE_DIR}/inc/ROOT/TArrowDS.hxx)
+  list(REMOVE_ITEM sources ${CMAKE_CURRENT_SOURCE_DIR}/src/TArrowDS.cxx)
+endif()
+
+ROOT_STANDARD_LIBRARY_PACKAGE(DataFrame
```

Comment:
New libraries shall be "scoped": `libROOTDataFrame` (to avoid clashes like libRIO and lbRGL etc). Could you update the library name?